apps/examples/testcase/ta_tc/task_manager/utc/utc_task_manager_main.c

```diff
@@ -106,17 +106,17 @@ static void utc_task_manager_start_n(void)
 	ret = task_manager_start(TM_INVALID_HANDLE, TM_NO_RESPONSE);
 	TC_ASSERT_EQ("task_manager_start_n", ret, TM_INVALID_PARAM);
 
+	ret = task_manager_start(TM_UNREGISTERED_HANDLE, 100);
+	TC_ASSERT_EQ("task_manager_start_p", ret, TM_FAIL_UNREGISTERED_TASK);
```

Comment:
This tc is for negative. Please change to TC_ASSERT_EQ("task_manager_start_n ~"

Reply:
I agree with this